client/src/Visit.test.tsx

```diff
@@ -159,15 +230,42 @@ describe('Visit', () => {
       } as AppConfigModel)
     );
 
-    const getChatThreadIdFromTeamsLinkSpy = jest.spyOn(GetTeamsMeetingLink, 'getChatThreadIdFromTeamsLink');
-    getChatThreadIdFromTeamsLinkSpy.mockReturnValue('threadId');
+    const getRoomsMeetingLinkSpy = jest.spyOn(GetMeetingLink, 'getRoomsMeetingLink');
+    getRoomsMeetingLinkSpy.mockImplementation(() => {
+      return {
+        roomId: 'mockRoomId'
+      } as RoomCallLocator;
+    });
+
+    const getRoomsUserIdSpy = jest.spyOn(GetMeetingLink, 'getRoomsUserId');
+    getRoomsUserIdSpy.mockReturnValue('mockParticipantId');
+  });
+
+  it('renders a generic error when rooms token throws an error', async () => {
```

Comment:
Since this only applies to `RoomsMeeting.tsx`, can we move this out of this test file and into `RoomsMeeting.test.tsx` ?